parse_node/parse_enum_declaration.ts

```diff
@@ -45,86 +45,52 @@ export const parseEnumDeclaration = (
     ".gd"
 
   return {
-    content: `const ${enumName} = preload("${resPath}").${enumName}`,
-    files: [
-      {
-        body: enumText.content,
-        filePath: fileName,
-      },
-    ],
+    content: enumText.content,
+    files: [],
   }
 }
 
 export const testEnumDeclaration: Test = {
   ts: `
 export enum MyEnum { A, B }
 
-export class Hello {
+export default class Hello {
   constructor() {
     print(MyEnum.A)
   }
 }
   `,
-
-  expected: {
-    type: "multiple-files",
-    files: [
-      {
-        fileName: "/Users/johnfn/MyGame/compiled/Hello.gd",
-        expected: `
+  expected: `
 class_name Hello
-const MyEnum = preload("res://compiled/Test_MyEnum.gd").MyEnum
-func _ready():
-  print(MyEnum.A)
-      `,
-      },
-
-      {
-        fileName: "/Users/johnfn/MyGame/compiled/Test_MyEnum.gd",
-        expected: `
 const MyEnum = {
   "A": 0,
   "B": 1,
-}`,
-      },
-    ],
-  },
+}
+func _ready():
+  print(MyEnum.A)
+  `,
 }
 
 export const testEnumDeclaration2: Test = {
   ts: `
 export enum TestEnum { 
-  A = "A", 
-  B = "B"
+  A = 1, 
```

Comment:
Why the change to 1, 2? We should handle string enums as well.

Reply:
Added back test case for strings, I have removed it by mistake.